tests/iio_common.c

```diff
@@ -307,14 +307,15 @@ struct iio_context * handle_common_opts(char * name, int argc,
 				fprintf(stderr, "uri options requires a uri\n");
 				return NULL;
 			}
-			backend = IIO_AUTO;
```

Comment:
It would be nice to have these changes (about context creation) in a separate commit.

Reply:
done. Sorry about that.